frontend/src/components/YesNoSelection.tsx

```diff
@@ -0,0 +1,92 @@
+import React, {FunctionComponent, ReactElement, useState} from "react";
+import styled from "styled-components";
+import {CORAL, SEA_FOAM} from "../constants/colors";
+import PurpleButton from "./PurpleButton";
+import {LikeFilled, DislikeFilled, CheckCircleFilled} from "@ant-design/icons";
+
+
+interface YesNoSelectionProps {
+  correctAnswer: boolean;
+}
+
+interface SelectionProp {
+  highlight: number,
+}
+
+const YesNoContainer = styled.div`
+  display: flex;
+  flex-grow: 1;
+  justify-content: end;
+  padding: 0em 0em 5em 0em;  
+`;
+
+const YesNoButton = styled(PurpleButton)`
+margin-left: 10px;
+margin-right: 10px;
+
+  :hover {
+  }
+
+
+    @media (max-width: 900px) {
+    width:100%;
+  }
+  
+  ${({highlight}: SelectionProp) => {
+  if (highlight === 1) {
+    return `
+    border: 5px solid ${SEA_FOAM}; 
+  `
+  } else if (highlight === 0) {
+    return `
+    border: 5px solid ${CORAL}; 
+  `
+  } else {
+    return `
+    border: 0px solid ${CORAL}; 
+  `
+  }
+  }
+}
+};
+  
+`;
+
+const YesNoSelection: FunctionComponent<YesNoSelectionProps> = ({correctAnswer}): ReactElement => {
+  let [selected, setSelected] = useState(-1);
+  const answer = correctAnswer ? 1 : 0;
+
+  return (
+      selected === answer ?
```

Comment:
I am thinking, instead of using a ternary here, would it be a bit easier to just let the YesNoButton take in a t/f for highlight? So like, if selected === answer && selected -> Yes; if selected !== answer && selected -> No; if !selected -> nothing

Reply:
makes sense, will make changes soon!

Reply:
hm actually I'm not seeing a clear way to do this with boolean, since there are 3 cases